src/components/skills-quiz/JobDescriptions.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {
+  useEffect, useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { InfoOutline } from '@edx/paragon/icons';
+import { logError } from '@edx/frontend-platform/logging';
+import { Skeleton } from '@edx/paragon';
+import { fetchJobPathDescription } from './data/service';
+import { DROPDOWN_OPTION_IMPROVE_CURRENT_ROLE, JOB_DESCRIPTION_DISCLAIMER } from './constants';
+
+const JobDescriptions = ({
```

Comment:
This component should be named `JobPathDescriptions` instead.

Reply:
Actually it renders description for a job plus description for a job path? so what you suggest?